modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -3260,6 +3283,74 @@ void ONNXImporter::parseQMatMul(LayerParams& layerParams, const opencv_onnx::Nod
     addLayer(layerParams, node_proto);
 }
 
+// A * B + C = Y, we require that the dimension of A is [m, k], and the dimension of B is [n, k].
+// And the dim of output Y is [m, n]
+void ONNXImporter::parseQGemm(LayerParams& layerParams, const opencv_onnx::NodeProto& node_proto)
+{
+    int ninputs = node_proto.input_size();
+    CV_Assert(ninputs == 8 || ninputs == 9);
+
+    layerParams.type = "InnerProductInt8";
+
+    if (constBlobs.find(node_proto.input(3)) == constBlobs.end())
+        CV_Error(Error::StsNotImplemented, "Variable weights is not supported");
+
+    Mat weights = getBlob(node_proto, 3);
+
+    if (!layerParams.get<int>("transB", 0))
```

Comment:
Do we support non-default values of `transA` or `alpha`?

Reply:
Yes, you're right.

Reply:
Hi, I did some research. And currently, we can do the same with `parseGemm`. Because the majority of `gemm` op was generated by the FC layer. And we always think the `transA` and  `alpha` are default values.

Reply:
Then it might be better if we had an assertion for that.

Reply:
Agree.